appeals/web/src/server/appeals/appeal-details/appeal-details.mapper.js

```diff
@@ -86,6 +95,37 @@ export async function appealDetailsPage(
 		}
 	};
 
+	const caseNotes = await Promise.all(
```

Comment:
It might be worth moving this into a separate function within the same file just to reduce the length of `appealDetailsPage`'s body